ports/servo/browser.rs

```diff
@@ -258,6 +258,12 @@ impl Browser {
                 EmbedderMsg::ResizeTo(_browser_id, size) => {
                     self.window.set_inner_size(size);
                 }
+                EmbedderMsg::Alert(browser_id, _message, response_chan) => {
+                    if let Err(e) = response_chan.send(true) {
+                        warn!("Failed to send alert response: {}", e);
+                        self.event_queue.push(WindowEvent::CloseBrowser(browser_id));
```

Comment:
@paulrouget Previously the constellation would panic the pipeline if sending failed, this was the closest I got to...

Reply:
I see that we used to call `self.handle_send_error(pipeline_id, e)`. Having a mechanism to report a communication error with the pipeline, from the embedder, sounds important

@asajeffrey, we are introducing a communication between the embedder and the pipeline. Any recommendation how we could handle a send error? Should we let the constellation now or handle it in the embedder code?